tests/behat/features/Users/Roles.feature

```diff
@@ -20,31 +20,41 @@ Feature: Roles
     Then I should get a "200" HTTP response
     And I am on "/node/add/stanford_page"
     And I should not see "Layout"
+    Then I am on "/node/1/delete"
```

Comment:
This should probably be updated to "Then I am on the front page" and I click "edit" to we are not locked to node/1